v2/cmd/voucher_server/server.go

```diff
@@ -39,6 +39,10 @@ var serverCmd = &cobra.Command{
 
 		config.RegisterDynamicChecks()
 
+		if config.IsCloudRun() {
```

Comment:
Question: will anything else use this setting?

As written, I'm wondering if we can reduce the configuration surface by suggesting the client set [`require_auth = false` in the config.toml](https://github.com/grafeas/voucher/blob/7c4e841b85f4dd2934bdb62cad498b8524328dfb/config/config.toml#L29) in this case, instead of a new env var.

Reply:
I don't think so, but then we would need separate configs for cloud run and gke.